packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -405,15 +403,43 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
       },
       select: {
         userId: true,
+        attendees: {
+          select: {
+            email: true,
+          },
+        },
+      },
+    });
+    if (
+      eventType.rescheduleWithSameRoundRobinHost &&
+      eventType.schedulingType === SchedulingType.ROUND_ROBIN
+    ) {
+      hosts = hosts.filter((host) => host.user.id === originalRescheduledBooking?.userId || 0);
+    }
+    const attendees = originalRescheduledBooking?.attendees.map((attendee) => attendee.email);
+    const users = await prisma.user.findMany({
```

Comment:
```suggestion
    const attendeesAvailability = await prisma.user.findMany({
```

Or something similar. Only naming it users doesn't give any context to what the variable stores.

Reply:
Done